Update log-analytics-data-collector-api - signature string format example

kylecweeks opened the pull request:
Update signature string format to replace \n with `n for carriage return. \n results in a bad authorization header.

azurecla commented:
Hi __@jeffpatton1971__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@camsoper can you please verify the customer's proposed changes?

CamSoper commented:
@jeffpatton1971 Hey, Jeff, thanks for the edit!  I'm the maintainer of that article (though not the original author) and it seems like that fell through the cracks when I took over.  I sincerely apologize for your trouble with it.

I'll step through it today and verify your edits (I'm sure they're good) and we'll get it merged in. 

@ggailey777 I'll let you know today when you can approve the PR.

CamSoper commented:
Okay, so based on the experiment I ran yesterday, there are some discrepancies with how CDN and storage accounts work together.  I have escalated to the engineering team to get some clarification.  I'll keep you posted.

ggailey777 commented:
OK, thanks @camsoper.

CamSoper commented:
I heard back from engineering.  The documentation was technically correct *as is*, but it omitted an important clarification regarding the **Origin Path** parameter, which is what led to the confusion @jeffpatton1971 experienced (along with longer-than-normal edge node propagation, we think).

I've made the required clarification to the doc on the private repository.  This PR can be closed without merge.

@jeffpatton1971 Thank you for bringing this to my attention.

ggailey777 commented:
@jeffpatton1971 Thanks for your contribution to fixing Azure documentation. We normally like to accept public PRs from customers so that you can get credit for the fix, but in this case it sounds like a fix was already made by @CamSoper in our internal repo. Please don't hesitate to contribute to Azure docs in the future; we will try our best to use your submission next time. 

At this point, I will close this PR.

kylecweeks committed:
Update log-analytics-data-collector-api.md

Update signature string format to replace \n with `n for carriage return. \n results in a bad authorization header.
